packages/issue-utils/test/issue-utils.ts

```diff
@@ -122,3 +122,63 @@ describe('addOrUpdateIssue', () => {
     scopes.done();
   });
 });
+
+describe('closeIssue', () => {
+  beforeEach(() => {
+    nock('https://api.github.com/').get('/app').reply(200, {
+      id: 12345,
+      slug: 'release-please',
+      name: 'Release Please',
+    });
+  });
+  it('closes an existing issue', async () => {
+    const scopes = nock('https://api.github.com/')
+      .get(
+        '/repos/test-owner/test-repo/issues?state=open&creator=release-please[bot]'
+      )
+      .reply(200, [
+        {
+          number: 123,
+          title: 'my issue title',
+          body: 'my issue body',
+          labels: [],
+        },
+      ])
+      .patch('/repos/test-owner/test-repo/issues/123')
```

Comment:
Does it make sense to assert the patch request's body?

Reply:
Done.